applications/crossbar/doc/user_authentication.md

```diff
@@ -12,10 +12,10 @@ Provides an auth-token via user credentials
 
 Key | Description | Type | Default | Required | Support Level
 --- | ----------- | ---- | ------- | -------- | -------------
-`account_name` | The account name of the user | `string(1..128)` |   | `false` |  
+`account_name` | The account name of the user | `string(ABC..Z)` |   | `true` |  
```

Comment:
`string(1..128)` just notes that the account name can be up to 128 characters. It can contain alphanumeric values too so `A-Z` wouldn't be accurate anyway.

Additionally the tables are auto-generated from the JSON schemas so editing them here in markdown is incorrect - find the corresponding JSON schema and edit the "description".

Reply:
I understand, my reasoning was that it could be alphanumeric and it wasn't clear to us. I only read this part of the doc and should have done more research before submitting.